collection.go

```diff
@@ -613,8 +622,39 @@ func resolveKconfig(m *MapSpec) error {
 
 			internal.NativeEndian.PutUint32(data[vsi.Offset:], value)
 
-		default:
-			return fmt.Errorf("unsupported kconfig: %s", n)
+		default: // Catch CONFIG_*.
+			configs[n] = configInfo{
+				offset: vsi.Offset,
+				typ:    v.Type,
+			}
+		}
+	}
+
+	// We only parse kconfig file if a CONFIG_* variable was found.
+	if len(configs) > 0 {
+		scanner, err := kconfig.Find()
```

Comment:
What I meant earlier: this function doesn't return a `scanner` anymore, so calling this `f` or similar is clearer.